src/main/java/school/lemon/changerequest/java/FractionNumber/FractionNumber.java

```diff
@@ -0,0 +1,70 @@
+package school.lemon.changerequest.java.FractionNumber;
+
+/**
+ * Created by lbrdev on 04.01.2017.
+ * Project: oop.pr1
+ */
+public class FractionNumber {
+    private int dividend;
+    private int divisor;
+    private static int DEFAULT_DIVISOR = 1;
+    public static final FractionNumber ONE = new FractionNumber(1);
+    public static final FractionNumber ZERO = new FractionNumber(0);
+
+    public FractionNumber(int dividend, int divisor) {
+        if (divisor == 0) throw new IllegalArgumentException("divisor must not be zero");
+        if (dividend == 0) {
+            this.dividend = dividend;
+            this.divisor = DEFAULT_DIVISOR;
+        } else {
+            this.dividend = dividend;
```

Comment:
There is no need to assign these fields. They will be overridden on the next few lines.